girder_tech_journal/__init__.py

```diff
@@ -36,13 +36,52 @@
 from girder.models.user import User
 from girder.utility import mail_utils
 from girder.utility.model_importer import ModelImporter
+from girder.plugin import GirderPlugin
 from girder import events
 from girder_worker_utils.transforms.girder_io import GirderUploadToItem
 from tech_journal_tasks.tasks import processGithub, surveySubmission
 from . import constants
 from .models.journal import download_statistics, Journal
```

Comment:
`download_statistics` should actually be named `DownloadStatistics` (since it's a class) and it should live in its own file at `models/download_statistics.py`. You can be fearless about moving this.